code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -101,7 +101,7 @@
 	new /obj/item/circuitboard/machine/techfab/department/security(src)
 	new /obj/item/storage/photo_album/hos(src)
 	new /obj/item/clothing/suit/hooded/wintercoat/security/hos(src)
-	new /obj/item/storage/box/gunset/glock18_hos(src)
+	//new /obj/item/storage/box/gunset/glock18_hos(src) //SKYRAT EDIT CHANGE - HOS NERF
```

Comment:
```suggestion

```
This wasn't correctly marked as a modular edit anyway, so you don't need to comment it out, you can just remove the line.